### PR TITLE
added Breaking news app

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Closes:  #644

- **Title** : Breaking News App
- **Name:** :  Maheshwari Love
- **Idenitfy yourself:** :  GSSOC

### Describe the add-ons or changes you've made 📃
Added new project

### How Has This Been Tested? ⚙️

Just added new project in Nextjs technology

### Screenshots 📷

Screenshot 2024-07-25 165254

### Type of change ☑️

added new project 
What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney please review this pr

File: Next-JS-Projects/Intermediate/Breaking-News-App/assets/images/newsarticle_placeholder.jpg
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Intermediate/Breaking-News-App/assets/images/newsarticle_placeholder.jpg:1]: reduce the size of file or use any other format of image.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Create a directory `Advanced` and keep your project inside in it.

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney i will raise the pr again

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney all changes done...

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by Avdhesh-Varshney** [comment]
Please fill this survey form https://forms.gle/Wvsp9N8TDF1WPStq9

**Pull request merged**